src/workerd/server/workerd-api.c++

```diff
@@ -474,20 +474,20 @@ void WorkerdApi::compileModules(
             jsg::ModuleRegistry::Type::INTERNAL);
       }
       // Inject artifact bundler.
-      {
-        using ModuleInfo = jsg::ModuleRegistry::ModuleInfo;
-        using ObjectModuleInfo = jsg::ModuleRegistry::ObjectModuleInfo;
-        using ResolveMethod = jsg::ModuleRegistry::ResolveMethod;
-        auto specifier = "pyodide-internal:artifacts";
-        modules->addBuiltinModule(specifier,
-            [specifier = kj::str(specifier)](
-                jsg::Lock& js, ResolveMethod, kj::Maybe<const kj::Path&>&) mutable {
-          auto& wrapper = JsgWorkerdIsolate_TypeWrapper::from(js.v8Isolate);
-          auto wrap = wrapper.wrap(js.v8Context(), kj::none, ArtifactBundler::makeDisabledBundler());
-          return kj::Maybe(ModuleInfo(js, specifier, kj::none, ObjectModuleInfo(js, wrap)));
-        },
-            jsg::ModuleRegistry::Type::INTERNAL);
-      }
+      // {
+      //   using ModuleInfo = jsg::ModuleRegistry::ModuleInfo;
+      //   using ObjectModuleInfo = jsg::ModuleRegistry::ObjectModuleInfo;
+      //   using ResolveMethod = jsg::ModuleRegistry::ResolveMethod;
+      //   auto specifier = "pyodide-internal:artifacts";
+      //   modules->addBuiltinModule(specifier,
+      //       [specifier = kj::str(specifier)](
+      //           jsg::Lock& js, ResolveMethod, kj::Maybe<const kj::Path&>&) mutable {
+      //     auto& wrapper = JsgWorkerdIsolate_TypeWrapper::from(js.v8Isolate);
+      //     auto wrap = wrapper.wrap(js.v8Context(), kj::none, Artifacts::disabled());
+      //     return kj::Maybe(ModuleInfo(js, specifier, kj::none, ObjectModuleInfo(js, wrap)));
+      //   },
+      //       jsg::ModuleRegistry::Type::INTERNAL);
+      // }
```

Comment:
@jasnell How do I get the new module registry working with a `JSG_STRUCT` instead of a `jsg::Object` subclass?

Reply:
Off the top of my head I don't know. Need to look at it again. Will try to do so soon but if this needs to land like this for now leave a todo comment and I can look at it when I'm back from PTO

Reply:
Unfortunately it seems like the workerd tests go through this code I commented out so I'll have to figure it out before merging. But I suspect I can figure it out myself if I try long enough, and if not I'll just have to wait for you to get back.